developer_manual/digging_deeper/users.rst

```diff
@@ -4,64 +4,54 @@ User management
 
 .. sectionauthor:: Bernhard Posselt <dev@bernhard-posselt.com>
 
-Users can be managed using the UserManager which is injected from the ServerContainer:
+Users can be managed using the IUserManager service that can be :ref:`injected<dependency-injection>`.
 
 .. code-block:: php
+    :caption: lib/Service/UserService.php
+    :emphasize-lines: 8, 10-12
 
     <?php
-    namespace OCA\MyApp\AppInfo;
 
-    use \OCP\AppFramework\App;
-
-    use \OCA\MyApp\Service\UserService;
-
-
-    class Application extends App {
+    namespace OCA\MyApp\Service;
 
-        public function __construct(array $urlParams=array()){
-            parent::__construct('myapp', $urlParams);
+    use OCP\IUserManager;
 
-            $container = $this->getContainer();
+    class UserService {
+        private IUserManager $userManager;
 
-            /**
-             * Controllers
-             */
-            $container->registerService('UserService', function($c) {
```

Comment:
We have come such a long way 🥲

Reply:
Developers today don't know how hard our life was back in our days